### PR TITLE
Add licenses to real-time-enforcer policy files

### PR DESCRIPTION
**Pull request by hshin-g**
+ Add licenses to real time enforcer module policy files

### CODE REVIEW

File: modules/real_time_enforcer/files/policy/bigquery/common.rego [hunk 1]
```diff
@@ -1,3 +1,18 @@
+# Copyright 2019 The micromanager Authors. All rights reserved.
```
Comment (morgante) [inline modules/real_time_enforcer/files/policy/bigquery/common.rego]: Who are the micromanager authors?

If we want to reference a set of multiple authors, we need an authors file: https://g3doc.corp.google.com/company/teams/opensource/releasing/authors.md?cl=head

Comment (hshin-g) [reply modules/real_time_enforcer/files/policy/bigquery/common.rego]: AUTHORS file here:  https://github.com/cleardataeng/micromanager/blob/master/AUTHORS

Where should this file go?

Comment (morgante) [reply modules/real_time_enforcer/files/policy/bigquery/common.rego]: The submodule root.

Comment (hshin-g) [reply modules/real_time_enforcer/files/policy/bigquery/common.rego]: @morgante Do we need to include Forseti and CFT as authors as well?

Comment (hshin-g) [reply modules/real_time_enforcer/files/policy/bigquery/common.rego]: Added the AUTHORS file with micromanager authors.

Comment (morgante) [reply modules/real_time_enforcer/files/policy/bigquery/common.rego]: Also need `Google LLC` as an author.

**Pull request merged**